lib/spack/spack/cmd/install.py

```diff
@@ -41,6 +41,8 @@
 from spack.package import PackageBase
 
 description = "build and install packages"
+section = "basic"
+level = "short"
```

Comment:
Nah, I would keep `basic usage` the same. `building` would be the next section so they don't have to travel far to see what they want.